parlai/tasks/self_chat/worlds.py

```diff
@@ -69,4 +108,8 @@ def parley(self):
             agents[0].observe(validate(acts[1]))
 
         self.update_counters()
-        self.cnt += 1
+        self.turn_cnt += 1
+
+
+class InteractiveWorld(SelfChatBaseWorld):
```

Comment:
@jaseweston It is still called InteractiveWorld down here!

I updated the class itself to be named something more descriptive, given the import in line 10 `parlai/tasks/convai2/worlds.py`. My thought was, if we were to change the interactive behavior at some point later, the import in `parlai/tasks/convai2/worlds.py` would break somewhat unexpectedly. By naming the class something more unique, if we change its implementation we could grep for `SelfChatBaseWorld` to more easily check what assumptions are made elsewhere in the code.